api/app/controllers/mno_enterprise/jpi/v1/admin/app_answers_controller.rb

```diff
@@ -14,13 +14,14 @@ def create
 
     def app_answer_params
       # for an admin, the organization does not matter
-      organization_id = current_user.organizations.first.id
+      orga_relation = current_user.orga_relations.first
       params.require(:app_answer).permit(:description)
-        .merge(user_id: current_user.id, question_id: parent.id, organization_id: organization_id, app_id: parent.app_id)
+        .merge(reviewer_id: orga_relation.id, reviewer_type: 'OrgaRelation',
+               parent_id: parent.id, reviewable_id: parent.reviewable_id, reviewable_type: 'App')
     end
 
     def parent
-      @parent ||= MnoEnterprise::AppQuestion.find(params[:question_id])
+      @parent ||= MnoEnterprise::Question.find_one(params[:question_id])
```

Comment:
I don't think it is is necessary. As it is a find, it is going to replace a direct questions/id request to a more complicated `where`: questions/filter[id]=id&filter[reviewable_type]=App